doc/dev.rst

```diff
@@ -233,11 +233,22 @@ images. The former is generally preferred, as it lets you update only those
 base images that have actually changed (the ones that haven't will be
 re-tagged).
 
-GitHub tags
------------
+Issue labeling
+--------------
 
-What kind of issue is it?
-~~~~~~~~~~~~~~~~~~~~~~~~~
+We use the following labels (a.k.a. tags) to organize issues. Each issue (or
+stand-alone PR) should have label(s) from every category, with the exception
+of disposition which only applies to closed issues.
+
+.. note::
+
+   This scheme is designed to organize open issues only. There have been
+   previous schemes, and we have not re-labeled closed issues.
```

Comment:
I think a statement on who is encouraged to label issues would be good. Is the goal for the issue submitter to self-label, the internal team label, or just you, Reid?